bot.py

```diff
@@ -126,13 +121,14 @@ def main(self):
             _end = int(time.time())
             _tot = _end - _start
             _min = min(list_countdown) - _tot
+            print(f"list_countdown: {list_countdown}")
```

Comment:
There is no need to display the list_countdown output.

Reply:
I know its useless i did it for debug purposes